src/Microsoft.PowerShell.ConsoleGuiTools/ShowObjectTreeCmdletCommand.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using OutGridView.Models;
+
+namespace OutGridView.Cmdlet
+{
+    [Cmdlet("Show", "ObjectTree")]
```

Comment:
Not sure why, but lowercase is not working:

<img width="709" alt="image" src="https://github.com/PowerShell/GraphicalTools/assets/585482/d9ed7fa3-191b-4ac2-8272-457395c55faa">

Should we add an alias? e..g `sot`?

Reply:
You have a typo in your screenshot, you are missing hte second `t`

I have added an alias in b05aac7